moped-editor/src/layouts/DashboardLayout/NavBar/NavigationSearchInput.js

```diff
@@ -139,7 +139,9 @@ const useStyles = makeStyles((theme) => ({
 const NavigationSearchInput = ({ input404Class }) => {
   const classes = useStyles();
   const divRef = React.useRef();
-  let projectSearchQuery = new GQLAbstract(ProjectsListViewQueryConf);
+  const projectSearchQuery = React.useRef(
+    new GQLAbstract(NavigationSearchQueryConf)
+  );
```

Comment:
I went with a ref here because I kept running into an issue where the component kept holding on to the previous search input value in the `projectSearchQuery` GQLAbstract instance. It seemed unintentional, but I can update to keep that behavior if needed.

Without ref:
https://github.com/cityofaustin/atd-moped/assets/37249039/66fe43b4-5d34-49b7-9d51-41f80d55aa7a

With ref:
https://github.com/cityofaustin/atd-moped/assets/37249039/a341eca8-da09-46c2-9092-efe1be54bff7

Reply:
excellent—this feels right! (i watched the vides after my other comment, blelow)